service/controller/legacy/v28/version_bundle.go

```diff
@@ -8,31 +8,46 @@ func VersionBundle() versionbundle.Bundle {
 	return versionbundle.Bundle{
 		Changelogs: []versionbundle.Changelog{
 			{
-				Component:   "TODO",
-				Description: "TODO",
-				Kind:        versionbundle.KindFixed,
+				Component:   "kubernetes",
+				Description: "Update kubernetes to 1.14.3. More info here: https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.14.md",
+				Kind:        versionbundle.KindChanged,
+			},
+			{
+				Component:   "calico",
+				Description: "Update calico to 3.7.2. More info here: https://docs.projectcalico.org/v3.7/release-notes/",
+				Kind:        versionbundle.KindChanged,
+			},
+			{
+				Component:   "containerlinux",
+				Description: "Updated to 2079.5.1. More info here: https://github.com/coreos/manifest/releases/tag/v2079.5.1",
```

Comment:
Tense needs to be imperative here and below. When you review your own changes you can spot most of these issues.

Reply:
Right. Forgot it here.